server/events/command_runner.go

```diff
@@ -176,10 +180,12 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(logger logging.SimpleLogging,
 		return
 	}
 
-	err = c.PreWorkflowHooksCommandRunner.RunPreHooks(ctx)
-
-	if err != nil {
-		ctx.Log.Errorf("Error running pre-workflow hooks %s. Proceeding with %s command.", err, command.Plan)
+	if err := c.PreWorkflowHooksCommandRunner.RunPreHooks(context.TODO(), ctx); err != nil {
+		c.Logger.ErrorContext(context.TODO(), "Error running pre-workflow hooks", map[string]interface{}{
+			"err": err,
```

Comment:
can you add repository and pull num here? similar to here:
https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/events/vcs/instrumented_client.go#L346-L348

Eventually this can be added to ctx and ripped out from there automatically but for now we need this.

Reply:
yeah, my thinking was that it will be covered by the context, but you're right in the meantime we need a workaround.